REAMDE

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
LICENSE is a symlink to LICENSE.md, and README a symlink to README.md. I quite like it that way, but if you'd rather not, no worries...